src/applications/edu-benefits/5490/components/BenefitSelectionWarning.jsx

```diff
@@ -5,22 +5,36 @@ export default function BenefitSelectionWarning(chapter, relationship) {
   if (chapter === 'chapter33') {
     // Fry
     if (relationship === 'child') {
-      warningText = 'I understand that I am choosing to receive Fry Scholarship benefits instead of DEA benefits for which I am currently eligible based on the death of the qualifying individual on this application. I understand that even after this choice I will continue to be eligible for DEA benefits if the eligibility is based on the death of the qualifying individual on this application, based on a parent who has a permanent and total service-connected disability, or based on any other criteria as listed in 38 U.S.C. § 3501(a)(1).';
+      warningText = (<div>
+        I understand that I’m choosing to receive Fry Scholarship benefits instead of DEA benefits for which I’m currently eligible based on the death of the qualifying individual on this application. I understand that even after this choice I’ll continue to be eligible for DEA benefits if my eligibility is based on:
+        <ul>
+          <li>The death of the qualifying individual on this application, and they died before August 1, 2011, or</li>
+          <li>A parent who has a permanent and total service-connected disability, or</li>
+          <li>Any other criteria as listed in 38 U.S.C. § 3501(a)(1)</li>
+        </ul>
+        based on any other criteria as listed in 38 U.S.C. § 3501(a)(1).
+      </div>);
     } else {
       // spouse
-      warningText = 'I understand that I am choosing to receive Fry Scholarship benefits instead of any DEA benefits for which I am currently eligible. This includes DEA benefits based on the death of the qualifying individual on this application, based on the death of any other individuals not listed on this application, based on a spouse who has a permanent and total service-connected disability, or based on any other criteria as listed in 38 U.S.C. § 3501(a)(1).';
+      warningText = (<div>
+        I understand that I am choosing to receive Fry Scholarship benefits instead of any DEA benefits for which I am currently eligible. This includes DEA benefits based on the death of the qualifying individual on this application, based on the death of any other individuals not listed on this application, based on:
```

Comment:
Should these two "I am" instances also switch to "I'm"?

Reply:
Probably- we were only asked to review the child section content and decided to update the spouse content to have a bulleted list like the child section.